cypress/e2e/new-suggestion-instruments.cy.ts

```diff
@@ -0,0 +1,111 @@
+const toInstrumentsProgressButton = "new-suggestion-progress-bar-instruments"
+const instrumentsArea = "area-instruments"
+const toReviewButton = "to-review-button"
+const searchInstrumentInput = "search-instrument-input"
+
+const instrumentSearchList = "instrument-search-list"
+const instrumentEditList = "instrument-edit-list"
+const deleteButton = "delete-button"
+const instrumentSearchCloseButton = "instrument-search-close-button"
+const descriptionInput = "description-input"
+
+const addInstrumentItem = () => {
+  cy.data(searchInstrumentInput).type("a")
+  cy.data(instrumentSearchList).first().click()
+}
+
+describe("when creating new instrument suggestions for a suggestion", function () {
+  beforeEach(() => {
+    cy.login()
+    cy.visit("/suggestions/new")
+    cy.data(toInstrumentsProgressButton).click()
+  })
+
+  context("having no instruments added", () => {
+    it("should prevent the process to proceed further", function () {
+      cy.data(toReviewButton).should("be.disabled")
+    })
+  })
+
+  context("having instruments added", () => {
+    beforeEach(() => {
+      addInstrumentItem()
+    })
+
+    it("should allow the process to proceed", function () {
+      cy.data(toReviewButton).click()
+      cy.data(instrumentsArea).should("not.exist")
+    })
+  })
```

Comment:
I'd remove the context & move the addInstrument to the `it` since you only have one text in this content which. So beforeEach is not necessary either.

Reply:
Same for the other contexts which only contain one test